src/homework4/task2.py

```diff
@@ -0,0 +1,32 @@
+"""Города
+   Дан список стран и городов каждой страны. Затем даны названия городов.
+   Для каждого города укажите, в какой стране он находится.
+   Входные данные
+   Программа получает на вход кол-во стран N. Далее идет N строк,каждая строка
+   начинается с названия страны, затем идут названия городов этой страны.
+   В следующей строке записано число M, далее идут M запросов — названия
+   каких-то M городов, перечисленных выше.
+   Выходные данные
+   Для каждого из запроса выведите название страны,
+   в котором находится данный город.
+"""
```

Comment:
Brest can be in France and Belarus. your solution doesn't handle such situations